### PR TITLE
mon: rename ceph mon stat to ceph mon status

### PR DESCRIPTION
**Pull request by zhongyimao**
Signed-off-by: Mao Zhongyi <maozy.fnst@cn.fujitsu.com>
https://tracker.ceph.com/issues/24232

### CODE REVIEW

**Comment by zhongyimao** [comment]
@joscollin According to Vikhyat's description, he recommend we standardize the format of the command as follow:
ceph \<daemon-type\> status

1. `ceph osd status`
2. `ceph mon status` <- at present - `ceph mon stat`
3. `ceph mds status` <-- at present - `ceph fs status`
4. `ceph rgw status` - not present
5. `ceph rbd-mirror status` - not present

but stat and status is different. stat is print summary messages, status print more detailed and specified messages, so I'm not sure really we need to simply rename `stat` to `status`? 

I have two plans
1. keep `stat` and `status` at the same time(we add a new command `mon status` like `osd status`)
2. delete all the implemented `stat` (`ceph osd/mds stat`) and only keep `status` because it already contains enough message.

any good suggestions? @vumrao

**Comment by joscollin** [comment]
@zhongyimao Why can't we send an email to ceph-devel (larger audience), finalize it and then create a PR? I also think one tracker would be enough - add PRs to it.

**Comment by zhongyimao** [comment]
@joscollin Thanks, and I just realized that.

File: src/mon/MonCommands.h [hunk 1]
```diff
@@ -419,7 +419,7 @@ COMMAND("mon dump " \
 	"name=epoch,type=CephInt,range=0,req=false", \
 	"dump formatted monmap (optionally from epoch)", \
 	"mon", "r", "cli,rest")
-COMMAND("mon stat", "summarize monitor status", "mon", "r", "cli,rest")
+COMMAND("mon status", "summarize monitor status", "mon", "r", "cli,rest")
```
Comment (jecluis) [inline src/mon/MonCommands.h]: commands should not simply be removed. Before removing a command, it needs to be marked as DEPRECATED. If the consensus is that we should get rid of `mon stat`, please DEPRECATE it first.

**Comment by vumrao** [comment]
Sorry for the confusion, I did not check that we have ceph osd stat and ceph mon stat has the same purpose. I wanted a command which can give all monitor daemon status. Let me update the tracker. I completely agree with @jecluis. But for now, I think we do not have to rename this command we should get a new command - `ceph mon status`.

**Comment by jecluis** [comment]
Safe to close the PR then?

**Comment by zhongyimao** [comment]
@jecluis I got a lot from this discussion, thank you very much, I will close it.

**Pull request closed without merging**